Include/arm_nnfunctions.h

```diff
@@ -432,7 +432,6 @@ int32_t arm_convolve_fast_s16_get_buffer_size(const cmsis_nn_dims *input_dims, c
  * @details
  *   - Supported framework : TensorFlow Lite Micro
  *   - The following constrains on the arguments apply
```

Comment:
Thanks for the contribution!
 Could you update the file version and date as well? https://github.com/ARM-software/CMSIS-NN#version--date

In this case, the version would be 11.2.1

Reply:
Of course, I just updated it https://github.com/ARM-software/CMSIS-NN/pull/20/commits/abc3b4ea8886c63c02ddbd033ecc8287a03e7f1f